src/pages/Post/PostAdd/PostAdd.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import './PostAdd.scss';
+import { useNavigate } from 'react-router-dom';
+
+const Post_add = () => {
+  const [content, setContent] = useState('');
+
+  const saveContent = event => {
+    setContent(event.target.value);
+  };
+
+  const navigate = useNavigate();
+  const goToBack = () => {
+    navigate('/post-list');
+  };
+
+  const handleSubmit = () => {
+    if (content.length < 1) {
+      alert('내용을 확인해주세요.');
+    } else {
+      fetch('', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          content: content,
+        }),
```

Comment:
이 내용을 다음처럼 작성할 수도 있습니다.
```suggestion
        body: JSON.stringify({
          content,
        }),
```